magentoerpconnect_bundle_import_childs/backend.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+#    Author Katja Matthes <katja.matthes at initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import openerp.addons.connector.backend as backend
+import openerp.addons.magentoerpconnect.backend as magento_backend
+
+magento1700_bundle_import_childs = backend.Backend(parent=magento_backend.magento1700,
+                                    version='1.7-bundle-import-childs')
```

Comment:
You shouldn't create a new backend version for a new feature (otherwise you will have to choose which version you want and it will allow only 1 feature at a time ;-))
Instead, you should rather use the `@magento(replacing=OtherConnectorUnit)` in order to modify the base behavior. When you do that, you must isolate only the behavior you must to replace in its own ConnectorUnit (e.g. do not replace ProductMapper but an isolated ProductBundleMapper). In fact there is already a pending basis for this which isolates the pieces: https://github.com/OCA/connector-magento/pull/95